cypress/component/CommentCard.cy.jsx

```diff
@@ -0,0 +1,75 @@
+import CommentCard from "../../src/components/TypesOfRecipeCards/CommentCard";
+import { testUser } from "../testData/testUser";
+
+const createTestUser = (isRecommend) => ({
+  ...testUser,
+  reviews: [
+    {
+      ...testUser.reviews[0],
+      isRecommend,
+    },
+  ],
+});
+
+const mountCommentCard = (user, review) => {
```

Comment:
Helper doesn't seem necessary. It doesn't really add much and I think `cy.mount(<CommentCard reviewer={user} review={review} />);` is easier to understand.